karma.conf-sauce.js

```diff
@@ -0,0 +1,70 @@
+/* jshint node: true */
+
+"use strict";
+
+// Browsers to run on Sauce Labs platforms.
+var sauceBrowsers = [
+  [ "chrome", "52", "Windows 10" ],
+  [ "chrome", "51", "Windows 10" ],
+  [ "firefox", "48", "Windows 10" ],
+  [ "firefox", "47", "Windows 10" ],
+  [ "microsoftedge", "13", "Windows 10" ],
+  [ "internet explorer", "11", "Windows 10" ],
+  [ "safari", "9", "OS X 10.11" ],
+  [ "safari", "8", "OS X 10.10" ]
+].reduce( function( object, platform ) {
+
+  // Convert "internet explorer" -> "ie".
+  var label = platform[ 0 ].split( " " );
+  label = (
+    label.length > 1 ? label.map( function( word ) { return word[ 0 ]; } ) : label
+  ).join( "" ) +
+  "_v" + platform[ 1 ];
```

Comment:
I have talked earlier about a test for this function, but if that's too annoying to create one (since we don't have a lot of tests as examples) then I guess we can live with it for now. It's all up to you.